bouncer/tests/lp_api_test.ts

```diff
@@ -98,8 +98,8 @@ async function testLiquidityDeposit(logger: Logger) {
   ).event;
 
   const liquidityDepositAddress = (
-    await lpApiRpc(logger, `lp_liquidity_deposit`, [testRpcAsset, 'InBlock'])
-  ).tx_details.response;
+    await lpApiRpc(logger, `lp_liquidity_deposit_v2`, [testRpcAsset, 'InBlock'])
```

Comment:
If we are not immediately deprecating the previous call, would it make sense to test both v1 and v2?